packages/bundler-plugin-core/src/utils/buildEndHelper.ts

```diff
@@ -0,0 +1,5 @@
+import { type Output } from "../types";
+
+export const buildEndHelper = (output: Output) => {
```

Comment:
l: IMO `xxxHelper` seems to have unclear naming for me. Maybe `updateOutputOnBuildEnd`?

another alternative is to have a output class that has these as public methods, and then can be serialized to an object when being sent.